scss/application.scss

```diff
@@ -24,11 +24,126 @@ body {
 }
 
 .page-content {
-  flex: 1;
   position: relative;
   overflow-y: auto;
+  flex-direction: column;
+  flex: 1;
+  padding: 50px;
+}
+
+.tech-card {
+  z-index: 3;
+  height: $tech-card-height + 10px;
+  width: $tech-card-width + 10px;
```

Comment:
If this is the `tech-card`, why are you using the `tech-card`'s variables but adding `10px` to them? :thinking: